src/main/java/teammates/logic/api/GateKeeper.java

```diff
@@ -396,19 +396,19 @@ private boolean isStudent() {
      * Verifies that the action is accessible when the user is either an instructor of the course, a student of the course
      * or his/her team member, or an admin.
      */
-    public void verifyAccessibleForCurrentUserAsInstructorOrTeamMemberOrAdmin(AccountAttributes account, String courseId,
+    public void verifyAccessibleForCurrentUserAsInstructorOrTeamMemberOrAdmin(String accountId, String courseId,
```

Comment:
`accountId` -> googleId. This is a more suitable name.

Reply:
Yes it is! I have updated it.